enterprise_access/apps/api/tasks.py

```diff
@@ -127,6 +130,50 @@ def send_notification_email_for_request(
     )
 
 
+@shared_task(base=LoggedTaskWithRetry)
+def send_cancel_email_for_pending_assignment(cancelled_assignment):
+    """
+    Send email via braze for cancelling pending assignment
+
+    Args:
+        cancelled_assignment: (string) the subsidy request uuid
+    """
+    learner_content_assignment_action = LearnerContentAssignmentAction(
+        uuid=cancelled_assignment.uuid, assignment=cancelled_assignment, action_type=AssignmentActions.CANCELLED
+    )
+
+    if braze_trigger_properties is None:
+        braze_trigger_properties = {}
+
+    braze_client_instance = BrazeApiClient()
+    lms_client = LmsApiClient()
+    enterprise_customer_uuid = learner_content_assignment_action.assignment_configuration.enterprise_customer_uuid
+    enterprise_customer_data = lms_client.get_enterprise_customer_data(enterprise_customer_uuid)
+    lms_user_id = learner_content_assignment_action.assignment.lms_user_id
+
+    try:
+        recipient = braze_client_instance.create_recipient(
+            user_email=cancelled_assignment.learner_email,
+            lms_user_id=cancelled_assignment.lms_user_id,
+        )
+        braze_trigger_properties["first_name"] = lms_user_id
+        braze_trigger_properties["organization"] = enterprise_customer_data.name
```

Comment:
`enterprise_customer_data` should be a plain dictionary here, you'll have to reference `name` as a key:
```python
blah = enterprise_customer_data['name']
```